website/docs/flags.md

```diff
@@ -21,7 +21,7 @@ flags { "flag_list" }
 | No64BitChecks         | Disable 64-bit portability warnings. Deprecated in Premake 5.0.0-beta8. Use `enable64bitchecks` API instead. |
 | NoBufferSecurityCheck | Turn off stack protection checks. Deprecated in Premake 5.0.0-beta8. Use `buffersecuritycheck` API instead. |
 | NoCopyLocal           | Prevent referenced assemblies from being copied to the target directory (C#) |
-| NoImplicitLink        | Disable Visual Studio's default behavior of automatically linking dependent projects. |
+| NoImplicitLink        | Disable Visual Studio's default behavior of automatically linking dependent projects. Deprecated in Premake 5.0.0-beta8. Use `implicitlink` API instead. |
```

Comment:
Do we have to have deprecated flags in the docs? Would it be better to delete it to avoid discovery and rely on the warning instead?